README.rst

```diff
@@ -23,6 +23,30 @@ This library provides a library called `thoth-storages
 for `PostgreSQL database <https://www.postgresql.org/>`__ as well as adapters
 for manipulating with `Ceph <https://ceph.io/>`__ via its S3 compatible API.
 
+Quick Start
+===========
+
+Pre-requisites:
+
+* make sure you have ``podman-compose`` installed
```

Comment:
podman and podman compose and I would add the command required (dnf install -y podman podman-compose)